Sources/Extensions/SwiftStdlib/StringProtocolExtensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  StringProtocolExtensions.swift
+//  SwifterSwift
+//
+//  Created by Max Härtwig on 11/26/17.
+//  Copyright © 2017 SwifterSwift
+//
+
+import Foundation
+
+public extension StringProtocol where Index == String.Index {
+    
+    /// SwifterSwift: The longest common suffix.
+    ///
+    ///        "Hello world!".commonSuffix(with: "It's cold!") = "ld!"
+    ///
+    /// - Parameter aString: The string with which to compare the receiver.
+    /// - Parameter options: Options for the comparison. See Apple's String Programming Guide for details on these options.
```

Comment:
We probably don't need to mention `See Apple's String Programming Guide for details on these options`